app-text/dblatex/files/dblatex-0.3.12-script.patch

```diff
@@ -0,0 +1,15 @@
+diff -Naur dblatex3-0.3.12-orig/scripts/dblatex dblatex3-0.3.12/scripts/dblatex
+--- dblatex3-0.3.12-orig/scripts/dblatex	2020-11-29 09:50:08.000000000 -0800
++++ dblatex3-0.3.12/scripts/dblatex	2024-06-09 16:42:31.071372334 -0700
+@@ -1,9 +1,8 @@
+-#! /usr/bin/env python3
++#!/usr/bin/python3
+ import sys
+ import os
+ 
+-package_base = os.path.join(os.path.dirname(sys.argv[0]), "..")
+-sys.path.insert(0, os.path.realpath(os.path.join(package_base, "lib")))
++package_base = r"/usr/share/dblatex"
```

Comment:
How does this handle EPREFIX?

Reply:
You may also consider prefix.eclass:

```bash
inherit prefix

src_prepare() {
    default

    # replace @GENTOO_PORTAGE_EPREFIX@ with the value of EPREFIX
    eprefixify scripts/dblatex 
}
```

```
package_base = r"@GENTOO_PORTAGE_EPREFIX@/usr/share/dblatex"
```

Reply:
TIL!

Reply:
Turns out the build process weirdly uses the existing script, so it cannot be patched in place.

I decided to go the way Debian went, but creating a Gentoo specific script in contrib but also patching it for EPREFIX.